roles/privacyidea/tasks/create_admin.yml

```diff
@@ -0,0 +1,23 @@
+---
+# The following admin users are defined in policies from admin2go:
```

Comment:
is this again admin2go or from the dump?

Reply:
You are right, it must be a dump file